pkg/host/internal/network/network.go

```diff
@@ -429,3 +429,44 @@ func (n *network) GetPciAddressFromInterfaceName(interfaceName string) (string,
 	log.Log.V(2).Info("GetPciAddressFromInterfaceName(): result", "interface", interfaceName, "pci address", pciAddress)
 	return pciAddress, nil
 }
+
+func (n *network) DiscoverRDMASubsystem() (string, error) {
+	log.Log.Info("DiscoverRDMASubsystem(): retrieving RDMA subsystem mode")
```

Comment:
minor: Please remove this log entry and the one on line 442. 
They would occur over 30 seconds as this function is part of the Discovery process.

Reply:
I removed this one.

if possible I would like to leave the other one on log level 2 for better debug let me know what do you think